client/README.md

```diff
@@ -228,19 +228,38 @@ Not you can build the binary itself:
 sbuild /path/to/.dsc -d plucky
 ```
 
-### Running autopkgtests locally in lxd
+### Running autopkgtests locally in VM
 
-To run autopkgtests, first set up the environment. It can be set up by
-running the following command (the distro can be different):
+Before proceeding please make sure you have QEMU installed on your
+system.
+
+To run autopkgtests, first we need to download the image (replace
+`plucky` with a corresponding release):
 
 ```sh
-autopkgtest-build-lxd ubuntu-daily:plucky/amd64
+autopkgtest-buildvm-ubuntu-cloud -r plucky -v \
+ --cloud-image-url http://cloud-images.ubuntu.com/daily/server
+```
+
+The image size may be too small, so you probably need to resize the
+disk and give it more storage:
+
+```
+qemu-img resize autopkgtest-plucky-amd64.img +15G
 ```
 
-Then run the autopkgtests in `lxd`:
+Then run the autopkgtests. The setup command adds more space to the
+`tmpfs` partition because autopkgtest for cargo use `/tmp` directory
+for building the package.
 
 ```sh
-autopkgtest /path/to/<package>_<version>_source.changes -- lxd autopkgtest/ubuntu/plucky/amd64
+autopkgtest \
+  --apt-upgrade \
+  --shell-fail \
+  --output-dir dep8-rust-hwlib \
+  --setup-commands="mount -o remount,size=10G /tmp" \
+    /path/to/<package>_<version>_source.changes \
```

Comment:
This can be . for the source tree. This would make the whole command copy paste friendly.

Reply:
Since this is a monorepo, just `.` may be confusing (also the changes file get generated in the parent dir). Or do you suggest mentioning that it should be done in the `client/` dir?

Reply:
Yeah, I meant the specific directory where it would work.

Reply:
I actually get this error if I pass the dir name instead of source.chages file

```
The following packages have unmet dependencies:
 autopkgtest-satdep : Depends: librust-hwlib-dev but it is not installable
                      Depends: hwctl but it is not installable
E: Unable to correct problems, you have held broken packages.
E: The following information from --solver 3.0 may provide additional context:
   Unable to satisfy dependencies. Reached two conflicting decisions:
   1. autopkgtest-satdep:amd64=0 is selected for install
   2. autopkgtest-satdep:amd64 Depends librust-hwlib-dev
      but none of the choices are installable:
      [no choices]
```

Reply:
I don't know why but I don't think this is crucial.